e2e-tests/trailing-slash/cypress/support/utils/trailing-slash.js

```diff
@@ -0,0 +1,14 @@
+export function assetPageVisits(pages) {
```

Comment:
typo, would have to change in other places as well
```suggestion
export function assertPageVisits(pages) {
```